### PR TITLE
[tr] Update content/tr/abstraction.md

### PR DESCRIPTION
**Pull request by symys**
### Describe your changes
Localize "Abstraction" page into Turkish

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/tr/abstraction.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Soyutlama (Abstraction)
+status: Completed
+category: Nitelik
+tags: ["fundamental", "", ""]
+---
+
+Bilişimde soyutlama, bir [hizmet](https://glossary.cncf.io/service/) kullanıcısından (bu bir program veya insan olabilir) ayrıntıları gizleyip, bir sistemi daha genelleyici ve kolay anlaşılır hale getiren bir temsildir.
```
Comment (aliok) [inline content/tr/abstraction.md:8]:
```suggestion
Bilişimde soyutlama, bir hizmet kullanıcısından (bu bir program veya insan olabilir) ayrıntıları gizleyip, bir sistemi daha genelleyici ve kolay anlaşılır hale getiren bir temsildir.
```

We removed links to non-localized pages in other documents. Let's go with that approach and add a note in the spreadsheet in `services.md` column, that there is a link to it from `abstraction.md`.

File: content/tr/abstraction.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Soyutlama (Abstraction)
```
Comment (rwxdash) [inline content/tr/abstraction.md:2]: One thing that I'm worried is that if the title is somewhat long, with the parentheses it'll be hard to read and probably won't look good. My suggestion is to keep the title in Turkish and include the English equivalent where we mention it first time in text, probably in the first paragraph.

Would love to hear your thoughts.

Comment (aliok) [reply content/tr/abstraction.md:2]: I like your suggestion, makes sense.

**Review by rwxdash — APPROVED** [review]

**Review by halil-bugol — APPROVED** [review]

**Review by aliok — APPROVED** [review]
/lgtm
/approve

**Pull request merged**